### ISSUE TITLE
Collection of SOGIE data for children

### ISSUE THREAD

**Issue by bchiaruttini** [mareinc/mare#732]
In new child registration/updates, add questions under gender:
-Pronouns used: with drop down options (she/her, he/him, they/them, other)  Selection viewable in Keystone
-Does child identify as part of the LGBTQ+ community? Options of Y/N/I don't know with comment box.  Selection and comments viewable in keystone
-(Non mandatory) If yes, is child comfortable with their identity being shared in their recruitment profile?  Options of Y/N/I don't know with comment box

**Comment by n-devr** [comment]
- [ ] add last two props at the bottom of the Child Information section
 - [ ] capture this data in the child registration email to MARE
 - [ ] add these fields to the child update form
 - [ ] ensure they get the yellow highlight
 - [ ] capture this data in the child update email

**Comment by n-devr** [comment]
- [ ] Set default values for does child identify and is child comfortable sharing identity to `Unknown`

**Comment by bchiaruttini** [comment]
Add to Child Listing Report LGBTQ+ Identified, with dropdown options of Y/N/IDK and can export in XLS

**Comment by bchiaruttini** [comment]
@n-devr Realizing we are going to have similar concern with mandating pronouns when saving child record. I assume this one is more difficult to have a default, unless able to easily tie to gender as a default (i.e female to she/hers).  Can we make this mandatory on reg form but not in keystone record? (I think you said no....)  Other ideas?

**Comment by bchiaruttini** [comment]
And finally, can we make Q about comfort sharing non-mandatory overall, as this isn't really relevant unless LGBTQ+ identified.

**Comment by n-devr** [comment]
@bchiaruttini all of these changes are in and ready for testing in `dev`.  I held off on pushing to `prod` bc of the open question on Pronouns and something I ran into with the Child Listing data.

**Pronoun Field**
As of now the Pronouns field is required on the registration form, but optional on the Child record.  This will work going forward as is - the only other option would be to add an `Unknown` value for this field like the other SOGIE fields and use that as the default.  Simple enough to leave it as-is or implement the `Unknown` option.

**Identity Data on Child Listing report**
When testing the data returned for LGBTQ identity queries I noticed that most of the children will have an empty value, which shows up on the form as `--`.  If a Child record is saved, the default value of `Unknown` will replace the empty value, but only after a record is saved.   Not sure if it's confused to have two different values that essentially mean `Unknown`.  I could convert all empty values to read as Unknown to make the data more consistent, or we could leave it as is to be able to differentiate between empty and `Unknown` if that provides any value - just lmk!
